types/k6/experimental/webcrypto.d.ts

```diff
@@ -91,14 +91,14 @@ export interface SubtleCrypto {
      *
      * To export a key, the key must have `CryptoKey.extractable` set to `true`.
      *
-     * @param format the format in which to export the key. Currently, only "raw" and "jwk" are supported.
+     * @param format the format in which to export the key.
      * @param key the key to export.
      * @throws {InvalidAccessError} - if the key is not extractable.
      * @throws {NotSupportedError} - if the format is not supported.
      * @throws {TypeError} - when trying to use an invalid format.
      * @returns A promise that resolves with the exported key.
      */
-    exportKey(format: "raw" | "jwk", key: CryptoKey): Promise<ArrayBuffer | JWK>;
+    exportKey(format: "raw" | "jwk" | "spki" | "pkcs8", key: CryptoKey): Promise<ArrayBuffer> | Promise<JWK>;
```

Comment:
does `Promise<Arraybuffer | JWK>` not work ?

Reply:
Ah, my bad I think I've just accidentally adjusted that

Reply:
Fixed with the latest force push